src/components/TopBar/TopBar.scss

```diff
@@ -5,6 +5,8 @@ $icon-size: rem(20px);
 $page-left-alignment-breakpoint-max: 1268px;
 $context-control-extra-spacing: rem(40px);
 $context-control-expand-after: 1400px;
+$shadow-when-scrolling: 0 1px 2px var(--p-shadow-from-direct-light);
+$shadow-easing: cubic-bezier(0.4, 0.22, 0.28, 1);
```

Comment:
Both of these variables (lines 8 and 9) look like good candidates for inclusion in the `tokens.ts` file so we know to include them in Polaris tokens once this work is complete.